harshal255/index.html

```diff
@@ -0,0 +1,313 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="
+    Hello Friends My Name is
+    Harshal Kahar
+    Welcome to my Portfolio...
+    I am a front-end Web developer as well as ux designer.
+    I am a 3rd Year Computer Engineering Student at L.D. College of Engineering in Ahmedabad.I am a front-end web developer,front-end Web developer as well as ux designer.. 
+    ">
+    <title>Harshal Kahar | Portfolio</title>
+    <link rel="icon" href="./IMG/Harshal1.jpg" type="image/icon type">
+
+    <!-- Links for Font & icons -->
+    <script src="https://kit.fontawesome.com/922f869533.js" crossorigin="anonymous"></script>
+    <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
+    <script nomodule src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
+    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
```

Comment:
Add Single Link for google font, you can add all font family in single link, you don't need to add all font style individually